README.md

```diff
@@ -85,3 +92,14 @@ await _hub.CloseAsync<MyMessage> (topic);
 // close all topcis
 await _hub.CloseAsync<MyMessage> (topic1);
 ```
+
+5. Error handling
+
+The library provides a way to handle exceptions that are thrown by the workers. The error worker will be called
+whenever an exception is thrown by a worker that is consuming a topic. It is up to the implementation to decide 
+if the message should be retried or not. 
+
+If we do not want to retry a message a worker can throw and exception, such exception will be added to a queue that 
+will be consumed by the error worker that was used when the topic was created. Each topic has its own error queue, that
```

Comment:
This paragraph seems repetitive of the above paragraph.
I think giving examples of how to handle the message with a retry and without a retry might make it more understandable